### ISSUE TITLE
Why do you have to screwdriver and crowbar windows twice anyways

### ISSUE THREAD

**Issue by Imagebard** [d3athrow/vgstation13#10483]
This is such a mess
Basically the method to unbolt windows is "crowbar screwdriver crowbar screwdriver", the problem is that the text for the first two actions repeat for the next two actions, which use the same tools, which is confusing.

I'm pretty sure this was reported before but searching "is:issue window" or "is:issue crowbar screwdriver" didn't pick up any results.

**Comment by Ephx1** [comment]
What? Isn't it just screwdriver-crowbar-screwdriver?

**Comment by kol1th** [comment]
What?

Window is screw-crow-screw.

**Comment by Exxion** [comment]
Yeah, it's definitely just screwdriver-crowbar-screwdriver, and the text each time is different

**Comment by Imagebard** [comment]
To move, not to rotate.

**Comment by ihadtoregisterforthis** [comment]
If you use the crowbar first you'll dent the glass.

**Comment by Exxion** [comment]
Yes, to move OR rotate there's only one crowbar.

**Comment by Skullyton** [comment]
1 screwdriver locks it in place, the crowbar and screwdriver after makes it airtight, or something like that.

**Comment by Exxion** [comment]
The crowbar and second screwdriver don't actually do anything other than make it take slightly longer to unfasten again, as far as I can tell

**Comment by AutismSpider** [comment]
putting the window on the grill makes it airtight or shitting down a full-window and screwdriver it down does the same thing

**Comment by Exxion** [comment]
Windows are airtight even if unanchored.

**Comment by Imagebard** [comment]
Exxion is right last time I checked.

**Comment by AutismSpider** [comment]
got screw it down or one tiny bump will fuck it up

**Comment by Shadowmech88** [comment]
I'm pretty sure unanchored windows can be moved by surges of air pressure, though.

**Comment by Exxion** [comment]
Yes, but they're still airtight when unanchored

I didn't exactly scour every inch of the code, but from what I did look at, the crowbar and second screwdriver do fuckall other than flavor text and making disassembly marginally harder

**Comment by ComicIronic** [comment]
It's to make breaking in slower.

**Comment by Exxion** [comment]
Doesn't do a good job, really, because none of the steps take any time

**Comment by ComicIronic** [comment]
And yet the process is still longer, because you have to switch hands and repeat the steps.

Either this has an impact and the reason is sound or it doesn't and why does this issue exist

**Comment by Imagebard** [comment]
Derailing my own PR here.
What if there was a way to reinforce the windows so you had to take 30 seconds dislodging them with a crowbar as a first step, or something of the such.It sounds like a small way to get Wardens to do something other than watching the buggy Camera console.
If you're wondering how to implement this, Silicate Sprayer.

>you have to switch hands

X is a default hotkey that switches hands automatically

**Comment by Exxion** [comment]
Well yes, the process is longer, but not significantly
The difference is significant for constructing or deconstructing a bunch of windows at once, to make a new room or repair a breach, but is insignificant for deconstructing one or two windows, as you would do when breaking in
